hare/hare_test.go

```diff
@@ -143,6 +143,7 @@ func TestHare_collectOutputAndGetResult(t *testing.T) {
 
 	pids := []types.ProposalID{types.RandomProposalID(), types.RandomProposalID(), types.RandomProposalID()}
 	set := NewSetFromValues(pids...)
+
```

Comment:
sorry for nitpicking, but changes in unrelated files should be avoided. it makes git history more confusing

Reply:
sure, fixed